bundles/org.openhab.binding.senseenergy/src/main/java/org/openhab/binding/senseenergy/internal/handler/SenseEnergyMonitorHandler.java

```diff
@@ -170,7 +168,7 @@ public void goOnline() {
             this.solarConfigured = apiMonitor.solarConfigured;
             apiMonitorStatus = getApi().getMonitorStatus(id);
             refreshDevices();
-        } catch (InterruptedException | TimeoutException | ExecutionException | SenseEnergyApiException e) {
+        } catch (Exception e) {
```

Comment:
Why changed this, we prefer the catch of explicit exceptions over the generic `Exception`